rust/agama-server/src/web/service.rs

```diff
@@ -85,8 +88,64 @@ impl MainServiceBuilder {
             .route("/auth", post(login).get(session).delete(logout));
 
         let serve = ServeDir::new(self.public_dir);
+
+        // handle the /po.js request
+        // the requested language (locale) is sent in the "agamaLanguage" HTTP cookie
+        // this reimplements the Cockpit translation support
+        async fn po(jar: CookieJar) -> impl IntoResponse {
```

Comment:
BTW this should be probably moved elsewhere. To `web/http.rs`? Any suggestion?

Reply:
Yes, `http.rs` looks good. And if it grows, it could have its own module.